api/models.py

```diff
@@ -75,3 +75,10 @@ class Node(ModelId):
     revision: Revision
     parent: Optional[PyObjectId]
     status: Optional[bool] = None
+    created: Optional[datetime] = Field(default_factory=datetime.utcnow)
+    updated: Optional[datetime] = Field(default_factory=datetime.utcnow)
+
+    @classmethod
+    def set_created(cls, obj, value):
```

Comment:
This should be the same as doing `obj.created = sometime`, right?

Reply:
Yes, it is the same. But we can not write Node specific logic to Database class. So, I created Node.set_created to do the same.